zerver/tornado/websocket_client.py

```diff
@@ -19,10 +19,10 @@
 
 
 class WebsocketClient:
-    def __init__(self, host_url, sockjs_url, sender_email, run_on_start, validate_ssl=True,
-                 **run_kwargs):
+    def __init__(self, host_url: str, sockjs_url: str, sender_email: str,
+                 run_on_start: Callable[..., None], validate_ssl: bool = True,
```

Comment:
For default arguments, we're keeping no space in the rest of the code, so we should have that here as well. It should be `validate_ssl: bool=True` instead.

There are some occurrences in `create_filter` method as well.